jobs/export_process/worker.go

```diff
@@ -39,27 +37,41 @@ func MakeWorker(jobServer *jobs.JobServer, app AppIface) model.Worker {
 
 		outPath := *app.Config().ExportSettings.Directory
 		ID := model.NewId()
-		app.SetID(ID)
-		exportFilename := ID + "_export.zip"
+		exportZipFilename := ID + "_export.zip"
+		exportLogFilename := ID + "_export.log"
 
-		rd, wr := io.Pipe()
+		zipReader, zipWriter := io.Pipe()
+		logReader, logWriter := io.Pipe()
+
+		errCh := make(chan *model.AppError, 2)
+		defer close(errCh)
```

Comment:
Explore why this isn't working inside the goroutine function. It should work.

Reply:
But now there are two goroutine function so I kept it outside